apps/dashboard/src/components/Navbar.tsx

```diff
@@ -0,0 +1,76 @@
+
+import Image from 'next/image'
+import Link from 'next/link'
+import { GiHamburgerMenu } from 'react-icons/gi'
+import userLogo from '../utils/user.png'
+import NavbarItems from './NavbarItems'
+const Navbar = () => {
+  const user = false
```

Comment:
Is there anyway for you to have the user variable be a React state which can get toggled on/off? I understand it's a basic prototype but it should be a bit more polished than setting the variable manually in the code.